stories/toast/props-table.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { ApiTable } from "../storybook-common/api-table";
+import { ApiTableSectionProps } from "../storybook-common/api-table/types";
+
+const DATA: ApiTableSectionProps[] = [
+    {
+        attributes: [
```

Comment:
This component extends props from HTMLDivElement. Hence this should be stated upfront. Refer to `src/stories/buttons/props-table.tsx` for more info